src/main/java/seedu/address/logic/commands/PendingQuestionCommand.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PENDINGQUESTION;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.StageManager;
+import seedu.address.model.Model;
+import seedu.address.model.course.Course;
+import seedu.address.model.person.PendingQuestion;
+import seedu.address.model.person.Student;
+
+/**
+ * Changes the pending question of an existing person in the address book.
```

Comment:
Can improve code documentation to be more relevant to the app, ie. person to student and address book to student list.